src/main/webapp/app/config/constants.tsx

```diff
@@ -510,6 +513,11 @@ export enum PAGE_ROUTE {
   NEWS = '/news',
   FDA_RECOGNITION = '/fda-recognition',
   LEVELS = '/levels',
+  DX = '/levels/diagnostics-levels',
+  PX = '/levels/prognostics-levels',
```

Comment:
If i remove "/levels" from the path, the "levels" tab will no longer be highlighted when a user is on this page. The website will no longer consider them to be under the "levels" tab in the navbar. considering that these are still levels, I would advice they remain in the "/levels" path as other levels can be added under here in the future for ease of reach.

Reply:
> I might misspell earlier, sorry about the confusion

I will remove "/levels" from the paths, although I would suggest it should be left like that, so that the "/levels" tab will be highlighted when a user is on any of the "levels" pages.Please let me know if you want me to add it back.